script.js

```diff
@@ -0,0 +1,647 @@
+        }else {
+            let str = display.innerHTML;
+            let res = str + '\r\n';
+            display.innerHTML = res;
+        } 
+    }
+    else if (event.key === 'Shift') {
+        CAPSROW1.classList.add('active');
+        row1.classList.add('active');
+        CAPS2.classList.add('active');
+        row2.classList.add('active');
+        CAPS3.classList.add('active');
+        row3.classList.add('active');
+        CAPS4.classList.add('active');
+        row4.classList.add('active');
+    }
+    else if (event.key === 'Control') {
+        if (row2lang.className === 'keyboard__row2Eng') {
+            row2lang.classList.add('active');
+            row2.classList.add('active');
+            row3lang.classList.add('active');
+            row3.classList.add('active');
+            row4lang.classList.add('active');
+            row4.classList.add('active');
+        }else {
+            row2lang.classList.remove('active');
+            row2.classList.remove('active');
+            row3lang.classList.remove('active');
+            row3.classList.remove('active');
+            row4lang.classList.remove('active');
+            row4.classList.remove('active');
+        }
+    }
+    else {
+        textArea.textContent += event.key;
+    }
+}
+
+document.onkeyup = function (event) {
+    if (event.key === 'Shift') {
+        CAPSROW1.classList.remove('active');
+        row1.classList.remove('active');
+        CAPS2.classList.remove('active');
+        row2.classList.remove('active');
+        CAPS3.classList.remove('active');
+        row3.classList.remove('active');
+        CAPS4.classList.remove('active');
+        row4.classList.remove('active');
+    }
+};
+/*
+const row2lang = document.querySelector('.keyboard__row2Eng');
+const row3lang = document.querySelector('.keyboard__row3Eng');
+const row4lang = document.querySelector('.keyboard__row4Eng');
+function changeLang () {
+    document.onkeydown = function (event) {
+        if (event.key === 'Control') {
+            if (row2lang.className === 'keyboard__row2Eng') {
+                row2lang.classList.add('active');
+                row2.classList.add('active');
+                row3lang.classList.add('active');
+                row3.classList.add('active');
+                row4lang.classList.add('active');
+                row4.classList.add('active');
+            }else {
+                row2lang.classList.remove('active');
+                row2.classList.remove('active');
+                row3lang.classList.remove('active');
+                row3.classList.remove('active');
+                row4lang.classList.remove('active');
+                row4.classList.remove('active');
+            }
+        }
+    };
+}
+changeLang();
+*/
+
+function enterWord () {
+    for(let k of key){
+        if (k != TAB) {
+            k.onclick = function(){
+                display.textContent += k.textContent;
+            }
+        }
+    } 
+}
+enterWord();
+
```

Comment:
try to create separate folder in source of the code. It calls "utils" or "shared" and in this folder you should create file "constants" where you put all rarely modified variables (as I have already mentioned). Also in this folder you can put file "event handlers" where you can  put all logic in addEventListeners. And also create file "utilityFunctions" or "functions" or "utilsFunctions" "helpers" or "helpersFunc" and there put some  small functions for the help.